artemis-cli/src/main/java/org/apache/activemq/artemis/cli/process/ProcessBuilder.java

```diff
@@ -29,17 +29,14 @@ public class ProcessBuilder {
    static ConcurrentHashSet<Process> processes = new ConcurrentHashSet<>();
 
    static {
-      Runtime.getRuntime().addShutdownHook(new Thread() {
-         @Override
-         public void run() {
-            for (Process p : processes) {
-               //               if (p.isAlive())
-               {
-                  p.destroy();
-               }
+      Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+         for (Process p : processes) {
+            //               if (p.isAlive())
+            {
+               p.destroy();
             }
```

Comment:
Since you are already there.. can you clean this up? just call p.destroy() and remove the if up there?

Reply:
I actually already amended the commit, and pushed it.

so I'm closing this as I already sent... nice once @jbertram